docs/tools/chrome-extension/pr-description.md

```diff
@@ -1,39 +1,39 @@
 ---
 id: pr-description
 title: 'PR Description'
-sidebar_label: 'PR Description'
+sidebar_label: PR Description
 keywords:
   - ai
   - pr descriptions
 ---
 
-This will help you to create a PR based on the configuration you have in the extension, it will enable you to create the summary based on either **Commit Messages** or **The diff between the files** or **both**, you can even set the length and the tone of the message using the same settings.
+The OpenSauced Chrome extension will help you create a PR based on the configuration you have in the extension. It will enable you to create the summary based on either commit messages, the difference between the files, or both. You can even set the message's length and tone using the same settings.
 
-_Note: This feature is not available on private repositories._
+_**Note**: This feature is not available on private repositories._
 
 ![generate pr description](../../../static/gif/pr-description.gif)
 
 You can use this settings to set the strictness and the tone of the generated AI, here is an explanation of those settings:
 
-- **Description** length:The output length.
-- **Temperature**: Is the similarity between the input text and the output, higher temperature mean more randomness, lower temperature means more strict to the input.
-- **Max Input length.**
-- **Description language:** right now this feature supports: English, Spanish, French, German, Italian, Portuguese, Dutch, Russian, Chinese, Korean.
-- **Description tone:** The tone of the output: Exciting, Persuasive, Informative, Humorous, Formal
-- **Description source:** The source of the description that our AI will use to get the output it can be the next:**Commit Messages** or **The diff between the files** or **both**
+- **Description Length**: The output length.
+- **Temperature**: The similarity between the input text and the output, higher temperature mean more randomness, lower temperature means more strict to the input.
+- **Max Input Length.**
+- **Description Language:** Right now, this feature supports: English, Spanish, French, German, Italian, Portuguese, Dutch, Russian, Chinese, Korean.
+- **Description Tone:** The tone of the output: Exciting, Persuasive, Informative, Humorous, Formal
+- **Description Source:** The source of the description that our AI will use to get the output it can be the next: **Commit Messages** or **The diff between the files** or **both**
```

Comment:
I can't test this out because the "orange OpenSauced" button doesn't appear in the PR form for me (see screen recording below). So I will give some feedback blindly. It's after the screen recording.

**Note**: I'm signing in to the extension and have enabled it in incognito mode. The button appears on the code explanation, but not in the PR form.

![OpenSauced button doesn't show up in PR form](https://github.com/open-sauced/docs/assets/45172775/a321239a-2c19-4aae-805c-c6b0a4485dd1)

## Feedback

-  What is the difference between "Description Length" and "Max Input Length"? We might want to give explanation in the "Max Input Length".
- > **Temperature**: The similarity between the input text and the output. Higher temperature means more randomness and lower temperature means stricter rules for the input.

  I believe output here is description as it's mentioned in the "Description Length"? But what is input? We might want to consider using the words as what it meant as explanation rather than "input" and "output" to avoid confuseness.
- We might as well want to explain the "Extensions Setting" (see screenshot below).

  ![opensauced ai settings](https://github.com/open-sauced/docs/assets/45172775/a1961541-cf31-4a0a-a051-8e58afa4bab6)

- Below is some suggestions for wordings adjustment.

```suggestion
- **Description Length**: The length of the description in characters.
- **Temperature**: The similarity between the input text and the output. Higher temperature means more randomness and lower temperature means stricter rules for the input.
- **Max Input Length.**
- **Description Language:** The languages that can be generated for the description. This feature currently supports English, Spanish, French, German, Italian, Portuguese, Dutch, Russian, Chinese, and Korean.
- **Description Tone:** The tone of the description. The available options are "Exciting", "Persuasive", "Informative", "Humorous", and "Formal".
- **Description Source:** The source that our AI will use to generate the PR description. It can be based on the commit messages, the difference between files, or both.
```

Reply:
I've adjusted the wordings as stated in the last point for clarity.
But what is "Max Input Length"?

Still need your thoughts for the other suggestion points.